src/extension.ts

```diff
@@ -65,7 +65,7 @@ export async function activate(context: vscode.ExtensionContext) {
 			} catch (e) {
 				// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 				// @ts-ignore
-				putLogEvent(e.stack);
+				CodeSyncLogger.error(e.stack);
```

Comment:
Can we add a message here as well along side the error stack?

Reply:
Same for other occurrences as well, I am skipping the comments for others.

Reply:
Good idea, will do. Thanks